tests/client.go

```diff
@@ -0,0 +1,18 @@
+package tests
+
+import (
+	"net/http"
+	"time"
+)
+
+type FaaSAuth struct {
```

Comment:
Call this  `Unauthenticated` because that is what it actually does. We will eventually reuse the CLI auth in the certifier, so having a "Unathenticated" would be nice because we can _always_ initialize an auth struct and this can be the default/fallback.

Reply:
Updated